src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -7194,57 +7267,6 @@ we will have to use something else.
                 tmp = ASR::make_LogicalConstant_t(al, loc, is_space,
                         ASRUtils::TYPE(ASR::make_Logical_t(al, loc, 4)));
                 return;
-            } else if (attr_name == "isalpha") {
```

Comment:
The CI fails because of the following changes, 
Why it needs to be modified?

Reply:
My bad @Thirumalai-Shaktivel. I made an error while creating this branch. These are somebody else's changes. I am opening a new PR as a change needs to be made regarding the method I use for checking `Const` types.